src/pudl/workspace/datastore.py

```diff
@@ -181,8 +181,8 @@ class ZenodoDoiSettings(BaseSettings):
     # eia_bulk_elec: ZenodoDoi = "10.5072/zenodo.1103572"
     epacamd_eia: ZenodoDoi = "10.5281/zenodo.7900974"
     # epacamd_eia: ZenodoDoi = "10.5072/zenodo.1199170"
-    epacems: ZenodoDoi = "10.5281/zenodo.8235497"
-    # epacems: ZenodoDoi = "10.5072/zenodo.1228519"
+    # epacems: ZenodoDoi = "10.5281/zenodo.8235497"
+    epacems: ZenodoDoi = "10.5072/zenodo.2237"
```

Comment:
Oooh, look it's mix-and-match production+sandbox DOIs and the number is sooo smol!